spec/wibox/widget/slider_spec.lua

```diff
@@ -12,6 +12,14 @@ describe("wibox.widget.slider", function()
         widget = slider()
     end)
 
+    describe("constructor", function()
+        it("applies arguments", function()
+            local widget = slider({ value = 10 })
```

Comment:
```suggestion
            widget = slider({ value = 10 })
```

This will prevent the `make check-qa` warning:

```
Checking spec/wibox/widget/slider_spec.lua        1 warning
spec/wibox/widget/slider_spec.lua:17:19: shadowing upvalue 'widget' on line 9
```